# Project Name
Quantifying the quality of single cells by analysis of counts and read depth ratio across the genome.

# Motivation
Cancer is one of the most prevalent diseases of the 21st century. The prompt and correct diagnosis of tumors is increasingly important to ensure accurate diagnosis and treatment. However, there can be many sources of noisy or spurious cells during singel cell sequencing. This project focuses on assigning each cell a score, based on how well each genomic segment correlates to a Poisson model. 

# Code Style
The code in this github is written in Python 3.8, and is executed in jupyter notebook. If you wish to replicate any of the experiments, please replicate the code inside a jupyter Environment, as some syntax shortcuts are used which are only processed by jupyter - other IDEs will raise errors.
Shortly, a script and python package of this code will become available.

# Installation?
Coming soon! (For now the studies can be replicated in jupyter notebook - can recommend fantastic IDEs such as Visual Studio Code which allow for an excellent interface with jupyter).

# Relevant Figures
Relevant figures will be uploaded here showing comparisons, and results.
